### ISSUE TITLE
interpret `--` as stdin

### ISSUE THREAD

**Issue by neonstalwart** [shelljs/shelljs#55]
in jshint/jshint#687, there's an interest in being able to use the jshint cli interface by passing `--` as a filename and having the input from stdin used as the contents of the file.  since jshint uses shelljs i was wondering if it would be possible to add this support to shelljs - it's probably useful to lots of other people who use shelljs too.

i'm not very familiar with shelljs but if someone could give me some guidance on how to approach this, i might take a stab at it.

**Comment by nfischer** [comment]
@neonstalwart Do you mean to modify shjs to allow the equivalent of this?

```
$ echo 'echo foo' | bash
```

If so, I think this will work as required:

```
$ echo "require('shelljs/global'); echo('foo');" | node
```

I'm closing this, since this seems to solve the issue. If this isn't sufficient, let me know, and we can reopen.

**Comment by neonstalwart** [comment]
I'm not sure if it's equivalent to what I meant but I've moved on quite a
lot since the time I opened this and no longer care.

On Sat, Jan 9, 2016, 16:36 Nate Fischer <notifications@github.com> wrote:

> Closed #55 <https://github.com/shelljs/shelljs/issues/55>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shelljs/shelljs/issues/55#event-510323491>.
>
-- 

--
ben...
